app/models/taxon_name.rb

```diff
@@ -413,4 +415,27 @@ def self.strip_author(name)
     name = name.gsub(/\s+[A-Z].*/, '')
     name.strip
   end
+
+  private
+
+  def english_lexicon_if_exists
+    en_lexicons = I18n.with_locale(:en) { I18n.t(:lexicons) }.values
+    translated_lexicons = I18n.available_locales.map { |loc| I18n.with_locale(loc) { I18n.t(:lexicons) } }
+    non_en_lexicons = (translated_lexicons.collect(&:values).flatten.uniq - en_lexicons).map!{ |l| l.downcase.strip }
+
+    if non_en_lexicons.include?(lexicon.downcase.strip)
```

Comment:
I'm not entirely sure *why* we allow the lexicon to be blank, but apparently we do, so guarding against that here will get you past the spec failures in Travis.

Reply:
A little digging found app/helpers/default_form_builder.rb:137 includes the option "Unknown" value blank which app/models/taxon_name.rb:181 in turns sets to nil. If you can't think of an immediate use case as to why nil is allowed, it makes me wonder if all nil values should default to `""` or `"Unknown"` with the appropriate db constraint `NOT NULL`. Still investigating though, so not sure. Maybe the data dumps will reveal some trends.